maps/chronosphere/upgrade_list.lua

```diff
@@ -214,7 +222,7 @@ function Public.upgrades()
       tooltip = {"chronosphere.upgrade_computer3_tooltip"},
       jump_limit = 25,
       cost = {
-        item1 = {name = "coin", tt = "item-name", sprite = "item/coin", count = 2000},
+        item1 = {name = "coin", tt = "item-name", sprite = "item/coin", count = 20000 * upgrade_coin_cost_scale}, --typo fix on coin cost I think?
```

Comment:
This upgrade has 10 tiers, so each costs 2000 -> for final 20 000

Reply:
Gotcha